code/game/sound.dm

```diff
@@ -42,7 +42,7 @@
  * * required_preferences - What preference is required to be on on the client, for the sound to play
  * * required_asfx_toggles - What toggles are required to be on on the client, for the sound to play
  */
-/proc/playsound(atom/source, soundin, vol as num, vary, extrarange as num, falloff_exponent = SOUND_FALLOFF_EXPONENT, frequency = null, channel = 0, pressure_affected = TRUE, ignore_walls = TRUE, falloff_distance = SOUND_DEFAULT_FALLOFF_DISTANCE, use_reverb = TRUE, required_preferences, required_asfx_toggles)
+/proc/playsound(atom/source, soundin, vol as num, vary, extrarange as num, falloff_exponent = SOUND_FALLOFF_EXPONENT, frequency = null, channel = 0, pressure_affected = TRUE, ignore_walls = FALSE, falloff_distance = SOUND_DEFAULT_FALLOFF_DISTANCE, use_reverb = TRUE, required_preferences, required_asfx_toggles)
```

Comment:
Do not change the function header, it's aligned with upstream except for the required preferences that are clearly marked as our thing, it is also expected that most sounds can be heard through walls, you can reduce the range of trivial things by using the appropriate extrarange defines to reduce how far they can go

Reply:
the default has been that sounds are not audible through walls, unless they're sufficiently loud, for years. i'm just restoring it to that.

Reply:
That's fine, it got changed, the whole system is different, and it's expected in this one that they are

Reply:
This is not addressed, as above, it is expected here that they traverse walls unless they are trivial and very quiet in frequency